lib/modules/manager/github-actions/extract.ts

```diff
@@ -7,7 +7,7 @@ import type { PackageDependency, PackageFile } from '../types';
 
 const dockerRe = regEx(/^\s+uses: docker:\/\/([^"]+)\s*$/);
 const actionRe = regEx(
-  /^\s+-?\s+?uses: (?<replaceString>['"]?(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>.+?)\s*['"]?(?:#\s+(?:renovate:\s+)?tag=(?<tag>.+?))?)\s*?$/
+  /^\s+-?\s+?uses: (:?(?<replaceString>['"]?(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>[\w.]+)?\s*['"]?\s*(:?#\s*(?:renovate:\s+)?tag=(?<tag>[v?\d+.]+))?)(:?.*)?)$/
```

Comment:
This looks really wrong now.
- Why there can be an addional `:` infront of the `replaceString`?
- Why is `currentValue` now optional? It's required.

I think this is what we need
```suggestion
  /^\s+-?\s+?uses: (?<replaceString>['"]?(?<depName>[\w-]+\/[\w-]+)(?<path>\/.*)?@(?<currentValue>\S+)['"]?(?:\s+#\s+(?:renovate:\s+)?tag=(?<tag>\S+))?)/
```
https://regex101.com/r/mKGVLK/1